test/e2e/case1_framework_deployment_test.go

```diff
@@ -16,18 +16,29 @@ import (
 )
 
 const (
-	case1ManagedClusterAddOnName  string = "governance-policy-framework"
-	case1ManagedClusterAddOnCR    string = "../resources/framework_addon_cr.yaml"
-	case1ClusterManagementAddOnCR string = "../resources/framework_clustermanagementaddon.yaml"
-	case1hubAnnotationMCAOCR      string = "../resources/framework_hub_annotation_addon_cr.yaml"
-	case1hubValuesMCAOCR          string = "../resources/framework_hub_values_addon_cr.yaml"
-	case1DeploymentName           string = "governance-policy-framework"
-	case1PodSelector              string = "app=governance-policy-framework"
-	case1MWName                   string = "addon-governance-policy-framework-deploy-0"
-	case1MWPatch                  string = "../resources/manifestwork_add_patch.json"
+	case1ManagedClusterAddOnName         string = "governance-policy-framework"
+	case1ManagedClusterAddOnCR           string = "../resources/framework_addon_cr.yaml"
+	case1ClusterManagementAddOnCRDefault string = "../resources/framework_clustermanagementaddon.yaml"
+	case1ClusterManagementAddOnCR        string = "../resources/framework_clustermanagementaddon_config.yaml"
+	case1hubAnnotationMCAOCR             string = "../resources/framework_hub_annotation_addon_cr.yaml"
+	case1hubValuesMCAOCR                 string = "../resources/framework_hub_values_addon_cr.yaml"
+	case1DeploymentName                  string = "governance-policy-framework"
+	case1PodSelector                     string = "app=governance-policy-framework"
+	case1MWName                          string = "addon-governance-policy-framework-deploy-0"
+	case1MWPatch                         string = "../resources/manifestwork_add_patch.json"
 )
 
-var _ = Describe("Test framework deployment", func() {
+var _ = Describe("Test framework deployment", Ordered, func() {
+	BeforeAll(func() {
+		By("Deploying the default governance-policy-framework ClusterManagementAddon to the hub cluster")
+		Kubectl("apply", "-f", case1ClusterManagementAddOnCRDefault)
+	})
+
+	AfterAll(func() {
+		By("Deleting the default governance-policy-framework ClusterManagementAddon from the hub cluster")
+		Kubectl("delete", "-f", case1ClusterManagementAddOnCRDefault, "--timeout=15s")
```

Comment:
Why time out? isn't `--grace-period=0 --force` better?

Reply:
I was following other tests here, but you're right--for cleanup a timeout isn't required since those were part of an E2E flow. But I think initially forcing deletion for cleanup is too aggressive. I'll remove the timeout.